test/testium-core.js

```diff
@@ -47,6 +47,9 @@ tap.test('Init against hello-world', function(t) {
         'Sets a _testium_ cookie');
     })
     .then(function() {
+      if (testium) {
+        testium.close();
+      }
       t.end();
     })
     .then(null, function(error) {
```

Comment:
Good point, actually `tap` accepted my PR to add promise support, so now they shouldn't be needed anymore. Will create a separate PR to clean that up.